searchform.php

```diff
@@ -6,7 +6,6 @@
 		<div class="SearchBar_Container">
 			<input class="SearchBar_Field" type="search" id="site-search" placeholder="<?php esc_html_e( 'Search the site &hellip;', 'hm-handbook' ); ?>" value="<?php the_search_query(); ?>" name="s"/>
 			<button class="SearchBar_Submit"><?php esc_html_e( 'Submit', 'hm-handbook' ); ?></button>
```

Comment:
It does need` type="submit"`. But the `submit_button` function is only for use in the admin.